charts/connect/CHANGELOG.md

```diff
@@ -11,6 +11,15 @@
 ## Security
 * A user-friendly description of a security fix. {issue-number}
 
+[//]: # (START/v1.17.0)
+# v1.17.0
+
+## Features
+* Add resources defaults to limit connect API's resource consumption. (#209, #211)
```

Comment:
I'm not sure how much this is a feature rather than a fix.

Reply:
If this is a fix, we can probably just bump the patch version since no new features are introduced. So we'd have 1.16.1

Reply:
If you feel like this is a fix rather than a feature I'm more than happy to switch it to a patch release.

The reason I leaned towards a minor is because it would be weird if a patch set those defaults and you had none before. 🤔

Reply:
Consulted the server definitions and it's okay to leave this as a minor increment.